test/runtests.jl

```diff
@@ -62,6 +62,11 @@ function finite_diff_mat_sym(f::Function, args::Tuple, i::Int, j::Int, k::Int, d
     return (f(pos_args...) - f(neg_args...)) / (2. * dx)
 end
 
+"Macro to test nesting of macros in @trace calls."
+macro insert_normal_call(x)
+    :(normal($(esc(x)), 0.0))
+end
+
```

Comment:
Optional: i.m.o. it would be more readable to move this (tiny) macro definition to right before the tests that use it (i.e. have it appear in both files) rather than having them refer to the function definition here.  Ex. [rationale](https://evgenii.com/blog/code-duplication-in-unit-tests/)